runtime/Java/src/org/antlr/v4/runtime/tree/IterativeParseTreeWalker.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * [The "BSD license"]
+ *  Copyright (c) 2012 Terence Parr
```

Comment:
I'm fine with everything here. Just tell me what, in your opinion, should be the copyright header, or if I can remove that completely? I must admit that I just copy/pasted the copyright notice from another file, trying to adhere to the code standards of this project, and not paying much attention to the header's content.